airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -428,31 +431,40 @@ def __init__(
         unmanaged_v_cpus: int | None = None,
         service_role: str | None = None,
         tags: dict | None = None,
-        max_retries: int | None = None,
+        poll_interval: int = 30,
+        max_retries: int = 120,
         status_retries: int | None = None,
         aws_conn_id: str | None = None,
         region_name: str | None = None,
+        deferrable: bool = True,
```

Comment:
`True` by default? I would not do that. That's a breaking change and I am not sure we want to enable the deferrable mode by default

Reply:
yeah oops didn't mean that